tsconfig.json

```diff
@@ -1,13 +1,15 @@
 {
   "compilerOptions": {
+    "composite": true,
```

Comment:
https://www.typescriptlang.org/tsconfig#composite
interesting. Didn't know about this option. Why did you turn it on? Does it make the build go faster?

Reply:
It was required for the tsconfig extends, didn't really looked a lot into the details 😄